src/components/subscriptions/addons/AddOnsListing.js

```diff
@@ -43,27 +54,101 @@ function AddOnsListing(props) {
         }
     }, [addonsData, selected]);
 
+    useEffect(() => {
+        setMultipleSelected(!!selected && selected.length > 1);
+    }, [selected]);
+
+    const { mutate: removeAddon, status: deleteAddonStatus } = useMutation(
+        deleteAddons,
+        {
+            onSuccess: () => {
+                announce({
+                    text: t("addonDelete"),
+                });
+                setSelected([]);
+                queryClient.invalidateQueries(AVAILABLE_ADDONS_QUERY_KEY);
+            },
+            onError: (error) => {
+                showErrorAnnouncer(t("deleteAddonError"), error);
+                setSelected([]);
```

Comment:
I'm not sure about clearing the user's selection on an error, because it may just be a temporary network or service error. So the user might want to try the delete again with the same selection (and a lot of times our error messages tell the user to please try again).

Reply:
That's a good point. I'll remove line 73.